packages/position/src/js/__specs__/index.spec.js

```diff
@@ -1,11 +1,13 @@
 import { cleanup, render, waitForElement } from 'react-testing-library'
 import React from 'react'
 
-import { above, below, leftOf, rightOf } from '../index.js'
+import * as positionFns from '../index.js'
```

Comment:
there was starting to be a lot of imports here so i collapsed it down. you'll see this is a couple places